### ISSUE TITLE
Centering tiles at MAP location makes procedure not Bayesian

### ISSUE THREAD

**Issue by ismael-mendoza** [prob-ml/bliss#387]
As discussed during the last meeting, we should think about centering at samples from the variational distribution of location at each tile predicted from the encoder. 

Currently not such a big deal since the current master branch galaxy autoencoder makes the procedure not Bayesian anyway. Maybe rethink after #339 is merged.

**Comment by jeff-regier** [comment]
Technically it's still a Bayesian procedure if you're using a MAP estimate for some latent variables---we're just not able to report the uncertainty. I agree though, we should switch to using samples from the variational distribution.

**Comment by jeff-regier** [comment]
Duplicate of #363
